### DISCUSSION TITLE
ESA Biomass Climate Change Initiative: Global datasets of forest above-ground biomass

### DISCUSSION THREAD

**Discussion by TomAugspurger** [microsoft/PlanetaryComputer#46] [Data request]
https://catalogue.ceda.ac.uk/uuid/5f331c418e9f4935b8eb1b836f8a91b8

> This dataset comprises estimates of forest above-ground biomass for the years 2010, 2017 and 2018. They are derived from a combination of Earth observation data, depending on the year, from the Copernicus Sentinel-1 mission, Envisat’s ASAR instrument and JAXA’s Advanced Land Observing Satellite (ALOS-1 and ALOS-2), along with additional information from Earth observation sources. The data has been produced as part of the European Space Agency's (ESA's) Climate Change Initiative (CCI) programme by the Biomass CCI team.

**Comment by milver** [comment]
@TomAugspurger there's a new version of the dataset. https://catalogue.ceda.ac.uk/uuid/af60720c1e404a9e9d2c145d2b2ead4e
This is a very important dataset that can help accelerate carbon storage and sequestration modeling.

**Reply by TomAugspurger** [reply]
Thanks for sharing that link.

**Comment by KBodolai** [comment]
v6.0 is now out too!: https://catalogue.ceda.ac.uk/uuid/95913ffb6467447ca72c4e9d8cf30501/
